File: 백준/Bronze/34946. 셔틀 탈래 말래 탈래 말래 애매하긴 해/셔틀 탈래 말래 탈래 말래 애매하긴 해.py
import sys
input=lambda:sys.stdin.readline().rstrip()
a,b,c,d=map(int,input().split())
if a+b>d and c>d:
    print("T.T")
else:
    if c>d:
        print("Shuttle")
    elif a+b>d:
        print("Walk")
    else:
        print("~.~")

File: 백준/Bronze/34946. 셔틀 탈래 말래 탈래 말래 애매하긴 해/셔틀 탈래 말래 탈래 말래 애매하긴 해.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int[] arr=Arrays.stream(scan.nextLine().split(" "))
        .mapToInt(Integer::parseInt).toArray();
        String st;
        if(arr[0]+arr[1]>arr[3] && arr[2]>arr[3])
            st="T.T";
        else {
            if(arr[2]>arr[3])
                st="Shuttle";
            else if(arr[0]+arr[1]>arr[3])
                st="Walk";
            else
                st="~.~";
        }
        System.out.print(st);
    }
}